Makefile

```diff
@@ -192,8 +172,16 @@ models/arwiki.reverted.gradient_boosting.model: \
 		--pop-rate "false=0.9648134044173649" \
 		--center --scale > $@
 
+stats/arwiki.reverted.gradient_boosting.model_info.txt: \
```

Comment:
how about putting this in `model_info/arwiki.reverted.md` so that it matches the other output patterns?  E.g. `models/...model` and `tuning_reports/...md`

Reply:
Agreed, will do.